index.html

```diff
@@ -0,0 +1,12 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```

Comment:
- [x] As stated the project requirement kindly make sure that you added A .css file that includes `one class` with styles for your header.
![Capture](https://user-images.githubusercontent.com/93598189/204540709-42bc6aff-5d1a-40e0-9418-5b08f5d372a1.PNG)

Reply:
Thank you, @Abenezer-Tilahun , for your rapid and thorough review comment regarding the CSS file class and what I missed on the header. I have added class to the header file.

Reply:
Hello @zewdu444 now you submit for re-review and it will be approved.

Reply:
@Abenezer-Tilahun, I'm sorry to inform you that I was unable to locate a re-review request option on the student dashboard. Could you perhaps direct me?